src/cf-vars.less

```diff
@@ -72,33 +72,33 @@
 // Color variables
 
 // body
-@text:                          darken(#895983, 15%);
+@text:                          #212121; // $color-base
 
 // a
-@link-text:                     #000080;
-@link-underline:                #000080;
-@link-text-visited:             lighten(@link-text, 5%);
-@link-underline-visited:        lighten(@link-underline, 5%);
-@link-text-hover:               darken(@link-text, 10%);
-@link-underline-hover:          darken(@link-underline, 10%);
-@link-text-active:              darken(@link-text, 15%);
-@link-underline-active:         darken(@link-underline, 15%);
+@link-text:                     #0071bc; // $color-primary
+@link-underline:                #0071bc; // $color-primary
+@link-text-visited:             #4c2c92; // $color-visited
+@link-underline-visited:        #4c2c92; // $color-visited
+@link-text-hover:               #205493; // $color-primary-darker
+@link-underline-hover:          #205493; // $color-primary-darker
+@link-text-active:              #205493; // $color-primary-darker
+@link-underline-active:         #205493; // $color-primary-darker
```

Comment:
I feel like we should break from what they've implemented currently to have a different active state. I suggest `$color-primary-darkest`.